flash/tabular/classification/data.py

```diff
@@ -311,3 +315,121 @@ def from_csv(
+        val_dict: Optional[Dict[str, List[Any]]] = None,
+        test_dict: Optional[Dict[str, List[Any]]] = None,
+        predict_dict: Optional[Dict[str, List[Any]]] = None,
+        target_formatter: Optional[TargetFormatter] = None,
+        input_cls: Type[Input] = TabularClassificationDictInput,
+        transform: INPUT_TRANSFORM_TYPE = InputTransform,
+        transform_kwargs: Optional[Dict] = None,
+        **data_module_kwargs: Any,
+    ) -> "TabularClassificationData":
+        """Creates a :class:`~flash.tabular.classification.data.TabularClassificationData` object from the given
+        dictionary.
+
+        .. note::
+            The ``categorical_fields``, ``numerical_fields``, and ``target_fields`` do not need to be provided if
+            ``parameters`` are passed instead. These can be obtained from the
+            :attr:`~flash.tabular.data.TabularData.parameters` attribute of the
+            :class:`~flash.tabular.data.TabularData` object that contains your training data.
+
+        The targets will be extracted from the ``target_fields`` in the dict and can be in any of our
+        :ref:`supported classification target formats <formatting_classification_targets>`.
+        To learn how to customize the transforms applied for each stage, read our
+        :ref:`customizing transforms guide <customizing_transforms>`.
+
+        Args:
+            categorical_fields: The fields (column names) in the dictionary containing categorical data.
+            numerical_fields: The fields (column names) in the dictionary containing numerical data.
+            target_fields: The field (column name) or list of fields in the dictionary containing the targets.
+            parameters: Parameters to use if ``categorical_fields``, ``numerical_fields``, and ``target_fields`` are not
+                provided (e.g. when loading data for inference or validation).
+            train_dict: The data to use when training.
+            val_dict: The data to use when validating.
+            test_dict: The data to use when testing.
+            predict_dict: The data to use when predicting.
+            target_formatter: Optionally provide a :class:`~flash.core.data.utilities.classification.TargetFormatter` to
+                control how targets are handled. See :ref:`formatting_classification_targets` for more details.
+            input_cls: The :class:`~flash.core.data.io.input.Input` type to use for loading the data.
+            transform: The :class:`~flash.core.data.io.input_transform.InputTransform` type to use.
+            transform_kwargs: Dict of keyword arguments to be provided when instantiating the transforms.
+            data_module_kwargs: Additional keyword arguments to provide to the
+                :class:`~flash.core.data.data_module.DataModule` constructor.
+
+        Returns:
+            The constructed :class:`~flash.tabular.classification.data.TabularClassificationData`.
+
+        Examples:
```

Comment:
![image](https://user-images.githubusercontent.com/19997320/167378835-19c005c0-8f74-45f8-b137-ec545579a88b.png)

This is how it looked before, so I am trying to change it to `Examples:` and if it looks better, maybe in a separate PR we can fix this? Or if I remember correctly, there was a PR by @Borda fixing this.